### PR TITLE
Introduce new variable to overwrite current root mount type

### PR DESCRIPTION
**Pull request by Mr-Pi**
plus removed useless set statement

### CODE REVIEW

File: templates/fstab.j2 [hunk 1]
```diff
@@ -6,8 +6,7 @@
 {%- if fstab_auto_root %}
 	{%- for mountpoint in ansible_mounts %}
 		{%- if mountpoint.mount=="/" %}
-			{%- set root={'options': mountpoint.options, 'source': mountpoint.device, 'type': mountpoint.fstype} %}
-			{{- fmtstr|format(root.source, '/', root.type, fstab_auto_root_options|default([root.options])|join(','),
+			{{- fmtstr|format(mountpoint.device, '/', fstab_auto_root_type|default(mountpoint.fstype), fstab_auto_root_options|default([mountpoint.options])|join(','),
```
Comment (SlothOfAnarchy) [inline templates/fstab.j2:9]: I think the list brackets in the default filter don't belong there. The variable already is a list. This way the filter creates a list of a list.

File: README.md [hunk 2]
```diff
@@ -10,20 +10,21 @@ A running linux system.
...
+| `fstab_auto_root`         | boolean                             |                `true`               | Auto generate a fstab entry for the root partion (based on current mount).                                        |
+| `fstab_auto_root_check`   | boolean                             |                `true`               | Set the fsck flag for auto generated root entry                                                                   |
+| `fstab_auto_root_dump`    | boolean                             |                `true`               | Set the dump/backup flag for the auto generated root entry                                                        |
+| `fstab_auto_root_options` | list of strings                     |      ___obtained from mount___      | Overwrite the current mount options                                                                               |
+| `fstab_auto_root_type`    | string                              |      ___obtained from mount___      | Overwrites root mount type (useful for network filesystems, for ex. if you changes a cifs to a nfs based backend) |
```
Comment (SlothOfAnarchy) [inline README.md]: Either "if one changes" or "if you change"

**Review by SlothOfAnarchy — CHANGES_REQUESTED** [review]

File: templates/fstab.j2 (hunk 1 above)
Comment (Mr-Pi) [reply templates/fstab.j2:9]: mountpoint.device is option obtained by ansible setup and contains a string. It's exact the same behavior like bevor.

**Comment by Mr-Pi** [comment]
@SlothOfAnarchy please rereview

File: templates/fstab.j2 (hunk 1 above)
Comment (SlothOfAnarchy) [reply templates/fstab.j2:9]: Ah, yes. I only looked up `fstab_auto_root_options` in your readme, which is a list.

**Review by SlothOfAnarchy — APPROVED** [review]

**Pull request merged**